spec/support/attribute_matcher.rb

```diff
@@ -1,11 +1,16 @@
-RSpec::Matchers.define :have_matching_attributes do |expected_hash, attribute_list|
-  match do |actual_hash|
-    attribute_list.each do |meth|
-      return false unless actual_hash.__send__(meth) == expected_hash.__send__(meth)
-    end
+# returns true if all the attributes in expected hash have the same values in actual_model.
+# Does not care if there are extra attributees in the model which are not in the hash
+#
+RSpec::Matchers.define :have_matching_attributes do |expected_hash|
+  match do |actual_model|
+    actual_hash = actual_model.attributes.symbolize_keys
+    actual_hash.keep_if { |key, _vallue| expected_hash.key?(key) }
```

Comment:
Typo in unused parameter, in both code blocks

Reply:
👍